README.md

```diff
@@ -1,194 +1,115 @@
 
-```dart
-final q = box.query(Entity_.number > 0)
-  .order(Type_.number)
-  .build();
-
-// ...
+Getting started
+---------------
 
-final qt = box.query(Entity_.text.notNull())
-  .order(Entity_.text, flags: Order.descending | Order.caseSensitive)
-  .build();
-```
+Add the following dependencies to start using ObjectBox and code generator.
 
-### Property Queries
+### Flutter mobile apps
 
-Instead of returning complete entities, with property queries only values or an aggregate of a property can be returned.
-Build a regular query with conditions as seen above, then turn it into a property query, e.g.:
+```yaml
+dependencies:
+  objectbox: ^0.10.0
+  objectbox_flutter_libs: any
 
-```dart
-// final query ...
-                            
-// Use distinct or caseSensitive to refine results.
-final textQuery = query.stringProperty(Note_.text)
-    ..distinct = true
-    ..caseSensitive = true;
-final texts = textQuery.find();
-textQuery.close();
-
-// Get aggregates, like min, max, avg, sum and count.
-final createdQuery = query.integerProperty(Note_.created);
-final min = createdQuery.min();
-createdQuery.close();
-
-// Set replaceNullWith to map null values.
-final scoreQuery = query.doubleProperty(Note_.score);
-final scores = scoreQuery.find(replaceNullWith: 0.0);
-scoreQuery.close();
-
-query.close();
+dev_dependencies:
+  build_runner: ^1.0.0
+  objectbox_generator: any
 ```
 
-### Streams
-
-Streams can be created from queries.
-Note: Dart Streams can be extended with [rxdart](https://github.com/ReactiveX/rxdart).
+* Install the packages: `flutter pub get`
+* XCode/iOS: under Architectures replace `${ARCHS_STANDARD)` with `arm64` (or `$ARCHS_STANDARD_64_BIT`). See [FAQ](#faq) for details.
 
-```dart
-    import "package:objectbox/observable.dart";
+### Dart CLI apps or Flutter desktop apps
 
-    // final store = ...
-    final query = box.query(condition).build();
-    final queryStream = query.stream;
-    final sub1 = queryStream.listen((query) {
-      print(query.count());
-    });
+```yaml
+dependencies:
+  objectbox: ^0.10.0
 
-    // box.put ...
+dev_dependencies:
+  build_runner: ^1.0.0
+  objectbox_generator: any
+```
 
-    sub1.cancel();
+* Install the packages: `(flutter|dart) pub get`
+* Install [objectbox-c](https://github.com/objectbox/objectbox-c) system-wide (use "Git bash" on Windows):
 
-    final stream = query.findStream(limit:5);
-    final sub2 = stream.listen((list) {
-      // ...
-    });
+  ```shell script
+  bash <(curl -s https://raw.githubusercontent.com/objectbox/objectbox-dart/main/install.sh)
+  ```
 
-    // clean up
-    sub2.cancel();
+* macOS: dart later complain that it cannot find the `libobjectbox.dylib`. You probably have to unsign the `dart`
+  binary - see [this dart issue](https://github.com/dart-lang/sdk/issues/38314#issuecomment-534102841) for details.
 
-    store.close();
-```
+* Windows: copy the downloaded `lib/objectbox.dll` to `C:\Windows\System32\` (requires admin privileges).
```

Comment:
Doesn't this lead to a versioning nightmare? E.g. all ObjectBox projects on that machine would need to use that version. And how to determine what version `objectbox.dll` is?

Reply:
Yes. That's the case with objectbox-c until 1.0 release and ABI stability.